client/my-sites/customer-home/main.jsx

```diff
@@ -186,14 +197,25 @@ class Home extends Component {
 								label={ translate( 'Add a page' ) }
 								iconSrc="/calypso/images/customer-home/page.svg"
 							/>
-							<ActionBox
-								onClick={ () => {
-									trackAction( 'my_site', 'write_post' );
-									page( `/post/${ siteSlug }` );
-								} }
-								label={ translate( 'Write blog post' ) }
-								iconSrc="/calypso/images/customer-home/post.svg"
-							/>
+							{ isStaticHomePage ? (
+								<ActionBox
+									onClick={ () => {
+										trackAction( 'my_site', 'write_post' );
+										page( `/post/${ siteSlug }` );
+									} }
+									label={ translate( 'Write blog post' ) }
+									iconSrc="/calypso/images/customer-home/post.svg"
+								/>
+							) : (
+								<ActionBox
+									onClick={ () => {
+										trackAction( 'my_site', 'manage_comments' );
+										page( `/comments/${ siteSlug }` );
+									} }
+									label={ translate( 'Manage comments' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **140** times:
`translate( 'Manage Comments' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).